packages/at_secondary_server/lib/src/verb/handler/enroll_verb_handler.dart

```diff
@@ -555,6 +558,11 @@ class EnrollVerbHandler extends AbstractVerbHandler {
       throw AtEnrollmentException(
           'Cannot revoke a ${enrollStatus.name} enrollment. Only approved enrollments can be revoked');
     }
+    if (operation == 'delete' && EnrollmentStatus.denied != enrollStatus) {
+      throw AtEnrollmentException(
+          'Cannot delete ${enrollStatus
+              .name} enrollments. Only denied enrollments can be deleted');
```

Comment:
Shouldn't we also allow "revoked" enrollments to be deleted?

Reply:
Since we've introduced the option to un-revoke enrollments, I think we should not delete them. Also, revoked enrollments have a TTL of 48 hours, so if not un-revoked they will anyway be deleted from the keystore after expiry.
What do you think @gkc?

Reply:
Giving the user the choice seems like the better option

Reply:
Makes sense, I will make the necessary changes and update tests.

Reply:
Changes to allow deletion of revoked enrollments and the updated tests have been added through [91525c0](https://github.com/atsign-foundation/at_server/pull/2095/commits/91525c0039cec2f56cdd5b4f4dbce121cbbae84c).